test/javascripts/turbolinks_test.coffee

```diff
@@ -1,210 +1,946 @@
 describe 'Turbolinks', ->
-
-  createTurboNodule = ->
-    $nodule = $("<div>").attr("id", "turbo-area").attr("refresh", "turbo-area")
-    return $nodule[0]
-
-  url_for = (slug) ->
+  server = null
```

Comment:
@TheMallen and I can't even run the tests in the browser period. We've been having to run teaspoon inline.